src/pudl/helpers.py

```diff
@@ -1049,34 +1066,15 @@ def drop_tables(engine: sa.engine.Engine, clobber: bool = False):
         None
     """
     md = sa.MetaData()
-    md.reflect(engine)
-    insp = sa.inspect(engine)
-    if len(insp.get_table_names()) > 0 and not clobber:
-        raise AssertionError(
-            f"You are attempting to drop your database at {engine} while clobber is set to {clobber}"
-        )
-    md.drop_all(engine)
-    conn = engine.connect()
-    conn.exec_driver_sql("VACUUM")
-    conn.close()
-
-
-def merge_dicts(list_of_dicts):
```

Comment:
Even if unused currently, can we imagine a use for this function? If so I'd suggest just keeping it.

Reply:
I can imagine it being useful, but whatever the actual use case was that we had seems to have been refactored away, and it's almost a one-liner... I feel like there should be an easy way to do this with a dictionary comprehension but it's not coming to mind immediately.

Reply:
Ah okay, as per Alex's suggestion

```py
{k: v for d in list_of_dicts for k, v in d.items()}
```